galaxy/importer/linters/__init__.py

```diff
@@ -68,16 +68,26 @@ def parse_id_and_desc(self, message):
             rule_desc = ' '.join(msg_parts[2:])
 
             error_id = msg_parts[1]
+            # REVIEW(cutwater): Flake8 is configured to return `E,F,W` rules.
+            #   Why is parsing rules prefixes list differs
+            #   from FLAKE8_SELECT_ERRORS.
```

Comment:
In deciding which flake8 issues to score at which severity, we did not want to map and maintain by individual rule, so we used E and W since they implied categorical severity, while the F pyflakes did not. Due in part to this complexity, we eventually decided to postpone scoring via flake8 (importer/loaders/base.py severity mappings do not have any flake8 matching), and keep it for ansible-lint (mapped to individual rules) and yamllint (mapped to categorical errors and warnings)